CHANGELOG.md

```diff
@@ -30,6 +30,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 - The `DebugCustom` derive is renamed to just `Debug` (gated now under a separate
   `debug` feature), and its semantics were changed to be a superset of `std` variant
   of `Debug`.
+- The `From` derive now uses `#[from(<types>)]` instead of `#[from(types(<types>))]`
+  and ignores field type itself.
```

Comment:
Hmm, I didn't realize this changed until I read this changelog entry. Sorry for not noticing earlier, because it's indeed in the PR description. But I don't think we should remove the `types` wrapper itself. This will make it hard for us to add new sub-attributes to the `from` attribute in a backwards compatible manner in the future, because although unlikely they might conflict with type names.

Reply:
To be clear this comment was about removing the `types` wrapper. "Ignoring the field type itself" I'm also not a huge fan of, but I think that's fine.